actionview/test/template/date_helper_test.rb

```diff
@@ -467,6 +469,22 @@ def test_select_year
     assert_dom_equal expected, select_year(2003, start_year: 2003, end_year: 2005)
   end
 
+  def test_select_year_with_empty_hash_value_and_no_start_year
+    expected = +%(<select id="date_year" name="date[year]">\n)
+    5.downto(0) { |i| expected << %(<option value="#{Date.current.year - i}">#{Date.current.year - i}</option>\n) }
```

Comment:
Can you make the expecation value static (fixed string)? Like other test cases do. It'll be a lot easier to understand.